### ISSUE TITLE
Avoid Banner-4 shuffling on page load with placeholder

### ISSUE THREAD

**Issue by leecalcote** [layer5io/layer5#3774]
### Current Behavior
On the homepage, banner, for will have its text, shuffled around on page load, while the short video is loaded on the right hand side. 

### Desired Behavior
Ideally, a placeholder is used or some other mechanism to ensure that the main text of the banner isn't jostled around while the video loads.

### Screenshots / Mockups
 
![Screenshot 2023-02-17 at 8 37 49 AM](https://user-images.githubusercontent.com/7570704/219685667-9292d2e1-4abf-47ee-b581-52d4654eeba8.png)

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.png" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by MrBehl06** [comment]
@leecalcote Please assign this issue to me?

**Comment by leecalcote** [comment]
@MrBehl06 thanks for asking. Given your prior track record 😬, will you first prototype and propose a solution?

**Comment by oneknucklehead** [comment]
Hey @leecalcote I believe we should implement skeleton loading for this

**Comment by leecalcote** [comment]
@oneknucklehead yes, an extremely lightweight placeholder, like an empty div that is configured under the same size constraints that the video is.

**Comment by MrBehl06** [comment]
@leecalcote I think We just need to add the margin-top and should decrease the max-width to 85% will solve this problem.
or else its position is relative we just add right margin.
 
<img width="1440" alt="Screenshot 2023-02-19 at 01 33 38" src="https://user-images.githubusercontent.com/101118926/219889265-98e45330-1848-46ad-996c-b4476dcf2e92.png">

**Comment by oneknucklehead** [comment]
> @oneknucklehead yes, an extremely lightweight placeholder, like an empty div that is configured under the same size constraints that the video is.

Is there an external link through which i can make get request for the video? Because that way, we can simply implement a skeleton loader. This also enhances the UX

**Comment by oneknucklehead** [comment]
hey @leecalcote 
Below Gif currently demonstrates my solution.
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/75200954/219945743-0f48e609-2426-4965-b3e5-3c2764b74e47.gif)

**Comment by Avi-88** [comment]
> hey @leecalcote Below Gif currently demonstrates my solution. ![ezgif com-video-to-gif](https://user-images.githubusercontent.com/75200954/219945743-0f48e609-2426-4965-b3e5-3c2764b74e47.gif) [ ![ezgif com-video-to-gif](https://user-images.githubusercontent.com/75200954/219945743-0f48e609-2426-4965-b3e5-3c2764b74e47.gif) ](https://user-images.githubusercontent.com/75200954/219945743-0f48e609-2426-4965-b3e5-3c2764b74e47.gif) [ ](https://user-images.githubusercontent.com/75200954/219945743-0f48e609-2426-4965-b3e5-3c2764b74e47.gif)



> hey @leecalcote Below Gif currently demonstrates my solution. ![ezgif com-video-to-gif](https://user-images.githubusercontent.com/75200954/219945743-0f48e609-2426-4965-b3e5-3c2764b74e47.gif) [ ![ezgif com-video-to-gif](https://user-images.githubusercontent.com/75200954/219945743-0f48e609-2426-4965-b3e5-3c2764b74e47.gif) ](https://user-images.githubusercontent.com/75200954/219945743-0f48e609-2426-4965-b3e5-3c2764b74e47.gif) [ ](https://user-images.githubusercontent.com/75200954/219945743-0f48e609-2426-4965-b3e5-3c2764b74e47.gif)



> @leecalcote I think We just need to add the margin-top and should decrease the max-width to 85% will solve this problem. or else its position is relative we just add right margin.
> 
> <img alt="Screenshot 2023-02-19 at 01 33 38" width="1440" src="https://user-images.githubusercontent.com/101118926/219889265-98e45330-1848-46ad-996c-b4476dcf2e92.png">

Hi @MrBehl06 , regarding your approach , rather than using width or margin to solve this issue which most probably won't be consistent across varying screen sizes its better to use a skeleton  as a placeholder for the video/image. @oneknucklehead's mockup looks promising , It just needs a little bit of tweaking to have the same dimensions as the media to be loaded. I would like to hear you guy's thoughts regarding the skeleton being an image similar to the media to be rendered or a classis pulsing gray div that is commonly used like below ,thanks

![skeleton](https://media.geeksforgeeks.org/wp-content/uploads/20200412121906/skeleton-loading.gif)

**Comment by Avi-88** [comment]
@oneknucklehead are you still up for this issue ?

**Comment by shirsho-roy** [comment]
@leecalcote can you assign this to me? I would like to work on this issue

**Comment by Savio629** [comment]
@shirsho-roy Any updates?

**Comment by shirsho-roy** [comment]
@Savio629  Working on it

**Comment by Savio629** [comment]
Any updates @shirsho-roy ?

**Comment by Savio629** [comment]
@shirsho-roy Are you still working on this issue?

**Comment by leecalcote** [comment]
Assignment seemingly abandoned. Opening for reassignment.

**Comment by saaagarsingh** [comment]
Hey @leecalcote @Savio629 I would love to work on this. 
Just need clarity that we need to have the space for video covered all the time so that text stays as it's position all the time even when the video is loading.

**Comment by ankushbhardwaj408** [comment]
https://github.com/layer5io/layer5/assets/93874580/097a32e2-eb15-4129-ae25-0d801fe29539

@leecalcote take a look at this.
